src/pages/Admin/Room/Edit/RoomEdit.scss

```diff
@@ -1,17 +1,15 @@
-import styled from "styled-components";
+@import "scss/constants.scss";
 
-export const Wrapper = styled.div``;
-
-export const ButtonsWrapper = styled.div`
-  display: flex;
-  justify-content: space-between;
-`;
-
-export const InputWrapper = styled.div`
+.input-wrapper {
```

Comment:
Following BEM we should probably have a 'root' class such as `.room-edit-modal` (which would be applied to the 'outermost container element', so probably the `Modal.Body` or similar, if it can have classes applied to it?), and then each of these would be an 'element' under that, so `.room-edit-modal__input` and `.room-edit-modal__buttons`.

Using scss, we can write/nest this as:

```scss
.room-edit-modal {
  &__input { /* add the scss here */ }

  &__buttons { /* add the scss here */ }
}
```

Further Reading:

- http://getbem.com/naming/
- https://css-tricks.com/the-sass-ampersand/
- https://sass-lang.com/documentation/style-rules/parent-selector